README.md

```diff
@@ -29,10 +29,10 @@ jobs:
 ```
 
 ## Optional Inputs
-- `helmfile-version` : helmfile version. Default `"v0.147.0"`.
-- `helm-version` : Helm version. Default `"v3.10.1"`
-- `kubectl-version` : kubectl version. Default `1.23.7`
-- `kubectl-release-date` : kubectl release date. Default `2022-06-29`
+- `helmfile-version` : helmfile version. Default `"v0.148.1"`.
+- `helm-version` : Helm version. Default `"v3.10.2"`
+- `kubectl-version` : kubectl version. Default `1.25.4`
+- `kubectl-release-date` : kubectl release date. Default `2022-11-09`
```

Comment:
Downloading kubectl gives 404 error.
Isn't the latest version 1.24.7?

https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html

Reply:
Hi, I will check and fix the error

Reply:
@kondoumh Yes, I realised after I created the PR, just trying to contribute